library/alloc/src/rc.rs

```diff
@@ -317,6 +317,8 @@ impl<T: ?Sized> !marker::Sync for Rc<T> {}
 
 #[stable(feature = "catch_unwind", since = "1.9.0")]
 impl<T: RefUnwindSafe + ?Sized> UnwindSafe for Rc<T> {}
+#[stable(feature = "rc_ref_unwind_safe", since = "1.56.0")]
```

Comment:
(Just noting that this will likely require a version bump because there's a release next week.)